src/app/admin/components/transformed-media/search-transformed-media-form/search-transformed-media-form.component.ts

```diff
@@ -53,12 +54,12 @@ export class SearchTransformedMediaFormComponent {
   @Input() courthouses: CourthouseData[] = [];
 
   form = this.fb.group({
-    requestId: ['', Validators.pattern(/^[0-9]*$/)],
-    caseId: [''],
+    requestId: ['', [Validators.pattern(/^[0-9]*$/), Validators.min(1), Validators.max(2147483647)]],
```

Comment:
Think we might be able to remove the pattern validation here now? It's showing this which is confusing.

![Screenshot 2025-01-07 at 11 41 58](https://github.com/user-attachments/assets/e4e054d5-74d9-4fa6-9d2e-cc388e294c5b)